.clang-format

```diff
@@ -0,0 +1,11 @@
+---
+Language: Cpp
+BasedOnStyle: Google
+
+ColumnLimit: 80
+NamespaceIndentation: Inner
+
+# Scaled by a factor of 2 such that the base indent is 4
+AccessModifierOffset: -3
```

Comment:
Why not -2?

Reply:
Discussion: https://github.com/dwavesystems/dwave-ocean-sdk/pull/103#discussion_r513567538